### PR TITLE
Use OAuth token to get repository list

### PR DESCRIPTION
**Pull request by arpit-saxena**
This way we can get all the repositories of the organization, including private repositories.

Only additional environment variable required is the personal access token of a member of the organization, and we can use a token for @devclubiitd

### CODE REVIEW

File: src/git/config.go [hunk 1]
```diff
@@ -31,4 +38,14 @@ func init() {
...
 		log.Fatal("GITHUB_SECRET is not present in environment, Exiting")
 	}
+
+	githubUserName = helper.Env("GITHUB_USERNAME", "None")
+	if githubUserName == "None" {
+		log.Fatal("GITHUB_USERNAME is not present in environment, Exiting")
+	}
+
+	githubAccessToken = helper.Env("GITHUB_ACCESS_TOKEN", "None")
+	if githubTAccessToken == "None" {
```
Comment (imperfect-fourth) [inline src/git/config.go]: typo in `githubTAccessToken`

File: src/git/config.go [hunk 2]
```diff
@@ -9,6 +9,13 @@ import (
 var (
 	// githubSecret is the secret used to verify that requests come from Github
 	githubSecret string
+
+	// githubToken is the OAuth token used to get details of GitHub private repositories
```
Comment (imperfect-fourth) [inline src/git/config.go]: `githubToken` -> `githubAccessToken`

**Review by imperfect-fourth — CHANGES_REQUESTED** [review]
Have you tested this to see if it works?

**Comment by arpit-saxena** [comment]
I've tested the core parts, the one in `src/git/git.go`. It does get our private repos as well.

**Review by imperfect-fourth — APPROVED** [review]
do a squash merge

**Pull request merged**